### DISCUSSION TITLE
"New language" Autolisp-VisualLisp

### DISCUSSION THREAD

**Discussion by Fraiddd** [community/community#51303] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, I'm not sure I'm posting in the right place...I'm attaching an XML file, taken from notepad++, which gives the details of the Autolisp-VisualLisp language, which is missing in your repository.
[Autolisp-VisualLisp.zip](https://github.com/community/community/files/11097841/Autolisp-VisualLisp.zip)
Thanks

**Comment by look** [comment, marked as answer]
@Fraiddd please file an issue in the Linguist repository using the [new language template](https://github.com/github/linguist/issues/new?assignees=&labels=Add+Language%2C+Good+First+Issue&template=new_language.md&title=).
